src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -209,6 +229,54 @@ var HTMLDOMPropertyConfig = {
   },
   DOMPropertyNames: {
   },
+  DOMMutationMethods: {
+    value: function(node, next) {
```

Comment:
`value` is shared by `<progress>` (and some other tag?), so this seems like a no-go. A progress with null value must remove the attribute.